packages/mds-native/api.ts

```diff
@@ -0,0 +1,101 @@
+/*
+    Copyright 2019 City of Los Angeles.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+        http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+ */
+
+import express from 'express'
+import { pathsFor, isValidProviderId, ValidationError, ServerError, AuthorizationError } from 'mds-utils'
+import logger from 'mds-logger'
+import db from 'mds-db'
+import { NextFunction } from 'connect'
+import { providerName } from 'mds-providers'
+import { asPagingParams, asJsonApiLinks } from 'mds-api-helpers'
+import { NativeApiResponse, NativeApiRequest, NativeApiGetEventsRequest, NativeApiGetEventsReponse } from './types'
+
+const NATIVE_API_VERSION = '0.0.1'
```

Comment:
We should be able to get this from `package.json`

Reply:
It's not the same thing. This is the API format version, not the code version.